internal/scheduler/internaltypes/node.go

```diff
@@ -149,3 +147,19 @@ func deepCopyLabels(labels map[string]string) map[string]string {
 	}
 	return result
 }
+
+func deepCopyStringToResourceList(m map[string]ResourceList) map[string]ResourceList {
```

Comment:
I *think* this (and the below function) are equivalent to `maps.clone`

Reply:
Fixed